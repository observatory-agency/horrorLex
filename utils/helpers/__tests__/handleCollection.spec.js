const handleCollection = require('../handleCollection');

describe('handleCollection', () => {
  it('should return the appropriate transformer and model for a collection', () => {
    expect(handleCollection('books')).toMatchSnapshot();
    expect(handleCollection('films')).toMatchSnapshot();
    expect(() => {
      handleCollection('somethingElse');
    }).toThrow();
  });
});
