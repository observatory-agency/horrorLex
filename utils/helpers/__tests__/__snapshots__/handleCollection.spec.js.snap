// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`handleCollection should return the appropriate transformer and model for a collection 1`] = `
Object {
  "model": Object {},
  "modelName": "publications",
  "transformer": [Function],
}
`;

exports[`handleCollection should return the appropriate transformer and model for a collection 2`] = `
Object {
  "model": Object {},
  "modelName": "publications",
  "transformer": [Function],
}
`;

exports[`handleCollection should return the appropriate transformer and model for a collection 3`] = `
Object {
  "model": Object {},
  "modelName": "films",
  "transformer": [Function],
}
`;
